modules/nw-ovn-ipsec-north-south-enable.adoc

```diff
@@ -18,16 +16,97 @@ After you apply the machine config, the Machine Config Operator reboots affected
 .Prerequisites
 
 * Install the {oc-first}.
+* You have installed the `butane` utility on your local computer.
+* You have installed the NMState Operator on the cluster.
 * You are logged in to the cluster as a user with `cluster-admin` privileges.
-* You have reduced the size of your cluster MTU by `46` bytes to allow for the overhead of the IPsec ESP header.
-* You have installed the `butane` utility.
 * You have an existing PKCS#12 certificate for the IPsec endpoint and a CA cert in PEM format.
+* You enabled IPsec in either `Full` or `External` mode on your cluster.
+* The OVN-Kubernetes network plugin must be configured in local gateway mode, where `ovnKubernetesConfig.gatewayConfig.routingViaHost=true`.
 
 .Procedure
 
-As a cluster administrator, you can enable IPsec support for external IPsec endpoints.
+. Create an IPsec configuration with an NMState Operator node network configuration policy. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/securing_networks/configuring-a-vpn-with-ipsec_securing-networks#configuring-a-vpn-with-ipsec_securing-networks[Libreswan as an IPsec VPN implementation].
+
+.. To identify the IP address of the cluster node that is the IPsec endpoint, enter the following command:
++
+----
+$ oc get nodes
+----
+
+.. Create a file named `ipsec-config.yaml` that contains a node network configuration policy for the NMState Operator, such as in the following examples. For an overview about `NodeNetworkConfigurationPolicy` objects, see link:https://nmstate.io/kubernetes-nmstate/[The Kubernetes NMState project].
++
+--
+.Example NMState IPsec transport configuration
+[source,yaml]
+----
+apiVersion: nmstate.io/v1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: ipsec-config
+spec:
+  nodeSelector:
+    kubernetes.io/hostname: "<hostname>" <1>
+  desiredState:
+    interfaces:
+    - name: <interface_name> <2>
+      type: ipsec
+      libreswan:
+        left: <cluster_node> <3>
+        leftid: '%fromcert'
+        leftrsasigkey: '%cert'
+        leftcert: left_server
+        leftmodecfgclient: no
```

Comment:
needs to change it to `leftmodecfgclient: false` , because if we deploy nncp yaml from web console ,  `leftmodecfgclient: no`  won’t work